notebooks/ch12/LR_Case_Study.md

```diff
@@ -0,0 +1,792 @@
+      <td>40.6</td>
+      <td>3</td>
+      <td>38</td>
+      <td>39.4</td>
+    </tr>
+    <tr>
+      <th>...</th>
+      <td>...</td>
+      <td>...</td>
+      <td>...</td>
+      <td>...</td>
+      <td>...</td>
+      <td>...</td>
+    </tr>
+    <tr>
+      <th>43</th>
+      <td>1</td>
+      <td>44.0</td>
+      <td>35.3</td>
+      <td>3</td>
+      <td>32</td>
+      <td>51.6</td>
+    </tr>
+    <tr>
+      <th>44</th>
+      <td>1</td>
+      <td>63.0</td>
+      <td>39.3</td>
+      <td>8</td>
+      <td>40</td>
+      <td>55.7</td>
+    </tr>
+    <tr>
+      <th>45</th>
+      <td>1</td>
+      <td>58.0</td>
+      <td>43.8</td>
+      <td>13</td>
+      <td>57</td>
+      <td>55.5</td>
+    </tr>
+  </tbody>
+</table>
+<p>44 rows × 6 columns</p>
+</div>
+
+
+
+How does our scatterplot look now without that outlier?
+
+
+```python
+# HIDDEN
+plt.scatter(drinking_data['Late births'], drinking_data['Cirrhosis'])
+plt.title("Late births vs. Cirrhosis")
+plt.xlabel("Late births")
+plt.ylabel("Cirrhosis");
+```
+
+
+![png](LR_Case_Study_files/LR_Case_Study_31_0.png)
+
+
+Much better!
+
+
+```python
+# HIDDEN
+plt.scatter(drinking_data['Wine'], drinking_data['Cirrhosis'])
+plt.title("Wine vs. Cirrhosis")
+plt.xlabel("Wine")
+plt.ylabel("Cirrhosis");
+```
+
+
+![png](LR_Case_Study_files/LR_Case_Study_33_0.png)
+
+
+This looks like a pretty strong linear relationship between wine and cirrhosis.
+
+
+```python
+# HIDDEN
+plt.scatter(drinking_data['Liquor'], drinking_data['Cirrhosis'])
+plt.title("Liquor vs. Cirrhosis")
+plt.xlabel("Liquor")
+plt.ylabel("Cirrhosis");
+```
+
+
+![png](LR_Case_Study_files/LR_Case_Study_35_0.png)
+
+
+There seems to be a somewhat linear relationship between liquor and cirrhosis, though it is definitely not as strong as that between wine and cirrhosis.
+
+## Linear Regression Model
+
+Since all variables seem to have a linear relationship with cirrhosis, we can just include all of them in our model. 
+
+Our model looks like this:
+
+$$
+f_\hat{\theta} (x) = \hat{\theta_0} + \hat{\theta_1} (Urban population) + \hat{\theta_2} (Late births) + \hat{\theta_3} (Wine) + \hat{\theta_3} (Liquor)
+$$
+
+Remember, the $\hat{\theta_0}$ is accounted for due to the bias column in our dataframe.
+
+Here are the functions we defined earlier in the multiple linear regression section, which we will use again:
+
+
+```python
+def linear_model(thetas, X):
+    '''Returns predictions by a linear model on x_vals.'''
+    return X @ thetas
+
+def mse_cost(thetas, X, y):
+    return np.mean((y - linear_model(thetas, X)) ** 2)
+
+def grad_mse_cost(thetas, X, y):
+    n = len(X)
+    return -2 / n * (X.T @ y  - X.T @ X @ thetas)
+```
+
+In order to use the above functions, we need `X`, and `y`. `thetas` is actually set to be an array of all 1's in the body of the `minimize` function. We can get these from our dataframe. Remember that `X` and `y` have to be numpy matrices in order to be able to multiply them with `@` notation.
```

Comment:
loos like thetas is set to all 0s not 1s. this is probably really confusing to the reader though, and minimize is hidden

I think it's fine if we leave out the sentence.

what does these in "We can get these" refer to?

nice protip with the @